index.html

```diff
@@ -163,10 +180,13 @@ <h2>About Myself</h2>
             <li><a class="m-button" href="#"><img class="social-icon" src="./icons/m-icon.png" alt="m button"></a></li>
         </ul>
         <button class="getResumeButton" type="button">Get my Resume</button>
```

Comment:
- [ ] Kindly, reduce the size of the button. Make sure you're using the correct text boldness & border-radius. It should match the Figma design.

Your design:
![image](https://user-images.githubusercontent.com/86069740/236355525-e77e9321-7bf0-4a31-bdde-d921228c3358.png)

Figma design:
![image](https://user-images.githubusercontent.com/86069740/236355575-fcf41c52-de26-4b9c-a36d-a2b892e2f05b.png)